scss/atoms/typography/_typography.scss

```diff
@@ -12,11 +12,11 @@
 body {
 	font: $normal #{ map-get( $defaultFont, font-size ) }/1.5 $sans;
 
-	@media ( max-width: 800px ) {
+	@media ( min-width: $viewport-xs ) {
```

Comment:
@cvrebert  umm the default starts at 14 px. Doesn't anything inside `@media ( min-width: $viewport-xs ) {` apply to screen sizes greater than $viewport-xs ?
 These variables are just breakpoints. with $viewport-sx defaulting to 400px.